cmake/helpers/CheckDependentLibraries.cmake

```diff
@@ -734,14 +734,25 @@ gdal_check_package(MONGOCXX "Enable MongoDBV3 driver" CAN_DISABLE)
 define_find_package2(HEIF libheif/heif.h heif PKGCONFIG_NAME libheif)
 gdal_check_package(HEIF "HEIF >= 1.1" CAN_DISABLE)
 
-# OpenJPEG's cmake-CONFIG is broken, so call module explicitly
-find_package(OpenJPEG MODULE)
+# OpenJPEG's cmake-CONFIG is broken with older OpenJPEG releases, so call module explicitly
+set(GDAL_FIND_PACKAGE_OPENJPEG_METHOD "MODULE" CACHE STRING "Method to use for find_package(OpenJPEG): CONFIG, MODULE or empty string")
+mark_as_advanced(GDAL_FIND_PACKAGE_OPENJPEG_METHOD)
+if(NOT GDAL_FIND_PACKAGE_OPENJPEG_METHOD STREQUAL "")
+    find_package(OpenJPEG ${GDAL_FIND_PACKAGE_OPENJPEG_METHOD})
+else()
+    find_package(OpenJPEG)
+endif()
```

Comment:
Isn't this the same as just ` find_package(OpenJPEG ${GDAL_FIND_PACKAGE_OPENJPEG_METHOD})`?